### DISCUSSION TITLE
Wert von letztem Digit

### DISCUSSION THREAD

**Discussion by reserve85** [jomjol/AI-on-the-edge-device#2531] [Q&A] [answered]
Hi, 

ich habe gestern das Update auf 15.3.0 durchgeführt und bemerkt, dass die RawValue immer beim letzten Digit ein value zu wenig anzeigt. An was könnte das liegen? Ist mir vorher noch nicht aufgefallen, aber ich kann mich auch täuschen.
![IMG_1091](https://github.com/jomjol/AI-on-the-edge-device/assets/111107925/a0f4b23b-c446-4947-b67f-bf948527f274)
![IMG_1090](https://github.com/jomjol/AI-on-the-edge-device/assets/111107925/318caa71-3fda-4be7-a224-7120e324ea6e)
![IMG_1093](https://github.com/jomjol/AI-on-the-edge-device/assets/111107925/4b59ef6b-4a8f-4442-b1fb-467f90f554f8)

**Comment by Slider0007** [comment, marked as answer]
Das sollte nichts mit dem Update zu tun haben.

Das liegt an der Rundung der Werte auf der Recognition Page. Der interne Algo hingegen schneidet nicht benötigte Nachkommastellen des ersten Digits ab. Ohne Extended Resolution wird nur die Ganzzahl, mit aktivem Extended Resolution die erste Nachkomma vom Algo betrachtet. Auch mit Extended Resolution kommt es noch zu einer Abweichung, aber dann nur bei der ersten Nachkommastelle.

Beispiel 1 (ohne Extended Resolution): Ein Eingangswert von 0.95 ergibt beim Algo 0 und bei der Anzeige auf der Recognition Page auf Grund Rundung 1.0.

Beispiel 2 (mit Extended Resolution): Ein Eingangswert von 0.95 ergibt beim Algo 09 und bei der Anzeige auf der Recognition Page 1.0

**Comment by reserve85** [comment]
Hi @Slider0007,

ja dachte mir schon, dass das nix mit dem Update zu tun hat, mir ist das nur zufällig direkt danach aufgefallen. Danke für die Erklärung, ich habe mir sowas in der Richtung schon gedacht. War nur verwundert wieso auf der Recognition Page der Value gestimmt hatte, aber aufgrund der Rundung macht das schon Sinn. Danke für die Erläuterung! 

Gruß Tobias
